### PR TITLE
added youtube video link to my talk.

### PR DESCRIPTION
**Pull request by narainsagar**
Hi @mashhoodr Can you please review and merge this? Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,6 @@ This repository is used for managing all our dev session resources and [rules](r
 | [Getting Started with Open Source Contribution](sessions/opensource-contribution/) | [Slides](https://slides.com/narainsagar/get-started-opensource-contribution), [Video](https://www.youtube.com/watch?v=xl8YOLCFKGU) | [Narain Sagar](https://github.com/narainsagar) | 12th July, 2017
 | [Jest Basics](sessions/jest/readme.md) | [Slides](https://slides.com/abdulmoiz/deck) | [Muhammad AbdulMoiz](https://github.com/abdulmoizeng) | 5th July, 2017 |
 | [What's new in Angular4](sessions/ng4/) | [Video](https://youtu.be/cKz6O6r9sCY), [Slides](https://docs.google.com/presentation/d/1j_-hN12xylHgbajVlik4GlEptAYNnkKNFGeBqykMPW4/edit#slide=id.p), [Live Demo](https://narainsagar.github.io/ng4-demo), [Code](https://github.com/narainsagar/ng4-demo) | [Narain Sagar](https://github.com/narainsagar) | 29th March, 2017
-| [Debugging Angular2+ Apps](sessions/ng-debugging/) | [Slides](https://slides.com/narainsagar/angular2-debugging) | [Narain Sagar](https://github.com/narainsagar) | 27th January, 2017
+| [Debugging Angular2+ Apps](sessions/ng-debugging/) | [Slides](https://slides.com/narainsagar/angular2-debugging), [Video](#) | [Narain Sagar](https://github.com/narainsagar) | 27th January, 2017
```
Comment (narainsagar) [inline README.md:31]: @mashhoodr ^^ this talk video is in office HDD - I remember, can we please upload this to our youtube channel? thanks

Comment (narainsagar) [reply README.md:31]: @sohaibnehal ^^

Comment (sohaibnehal) [reply README.md:31]: @narainsagar Maybe ping hassan for this.

Comment (narainsagar) [reply README.md:31]: Ok, As I'm not our slack channel anymore so can you please leave him a message to him, I'll also try to email / message on WhatsApp. Thanks.

**Pull request merged**